go/vt/throttler/throttler.go

```diff
@@ -224,6 +225,28 @@ func (t *Throttler) Throttle(threadID int) time.Duration {
 	return t.threadThrottlers[threadID].throttle(t.nowFunc())
 }
 
+// LastMaxLagNotIgnoredForTabletType returns the max of all the last replication lag values seen across all tablets of
+// the provided type, excluding ignored tablets.
+func (t *Throttler) LastMaxLagNotIgnoredForTabletType(tabletType topodata.TabletType) uint32 {
```

Comment:
I was thinking about why I don't like this function name.
- It's too long
- tabletType is an argument to the function so it doesn't need to be in the name
- it seems to be `MaxLast` vs `LastMax`
- The func name doesn't need to describe everything it is doing, that's what the comment is for.
- I wonder if even `Last` is required in the name at all

Reply:
Renamed as suggested below.